### ISSUE TITLE
Will Training code be available?

### ISSUE THREAD

**Issue by BrennoCaldato** [scape-research/SOSNet#2]
Hi, will the training code be available for reproducing the training with you loss?

Thanks

**Comment by vbalnt** [comment]
Hi, 

a training example with the SOS loss function will be added at some point, probably some time before the end of July. Will keep this open until then.

**Comment by willard-yuan** [comment]
Looking forward to access to the training code too.

**Comment by appleleaves** [comment]
Will Training code be available?

**Comment by valavanisleonidas** [comment]
Having the training code would be really good :)

**Comment by Smellly** [comment]
Will Training code be available?

**Comment by HorseCheng** [comment]
Will Training code be available?

**Comment by swjtuer0762** [comment]
The training code is here.[](https://github.com/yuruntian/SOSNet)

**Comment by swjtuer0762** [comment]
[](https://github.com/yuruntian/SOSNet)

**Comment by swjtuer0762** [comment]
Training codes are available [here](https://github.com/yuruntian/HyNet).

**Comment by swjtuer0762** [comment]
> Training codes are available [here](https://github.com/yuruntian/HyNet).

The author’s reply can be seen [here](https://github.com/yuruntian/SOSNet).
